packages/app-content-pages/src/shared/components/Head/Head.spec.js

```diff
@@ -1,41 +1,21 @@
-import { shallow } from 'enzyme'
+import NextHead from 'next/head'
+import sinon from 'sinon'
+import { render, screen } from '@testing-library/react'
 
 import Head from './Head'
 
-let wrapper
-
 describe('Component > Head', function () {
-  before(function () {
-    wrapper = shallow(<Head />)
-  })
-
-  it('should render without crashing', function () {
-    expect(wrapper).to.be.ok()
-  })
-
-  it('should render a meta `og:url` tag', function () {
-    const tag = wrapper.find(`meta[property='og:url']`)
-    expect(tag).to.have.lengthOf(1)
-  })
-
-  it('should render a meta `og:title` tag', function () {
-    const tag = wrapper.find(`meta[property='og:title']`)
-    expect(tag).to.have.lengthOf(1)
-  })
+  const stub = sinon.stub(NextHead, 'default').callsFake(({ children }) => <>{children}</>)
```

Comment:
Nice! I've been thinking of doing something like this for `@visx` components too.